public/Lab-Files/Lesson04/Challenges/Team-Challenge-2/index.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Green Team - Home</title>
+    <link rel="stylesheet" href="css/styles.css" type="text/css" />
+</head>
+
+<body>
+    
+            
+    <nav>
+        <div class="logo"><img src="img/logo.png" alt="pic of logo"/></div>
+        <ul class="button">
+            <li><img src="img/home-icon.png" alt="pic of home icon"/>Home</li>
+            <li><img src="img/deal-icon.png" alt="pic of deals"/>Travel Deals</li>
+            <li><img src="img/schedule-icon.png" alt="pic of schedule"/>Flight Schedule</li>
+            <li><img src="img/login-icon.png" alt="pic of login"/>Register/Login</li>
+        </ul>
+    </nav>
+    
+    <main>
+        <section>
+            <ul class="formbutton">
+                <li><img class="plane" src="img/plane-icon.png" alt="pic of plane"/></li>
+                <li><img class="hotel" src="img/hotel-icon.png" alt="pic of hotel"/></li>
+                <li><img class="car" src="img/car-icon.png" alt="pic of car"/></li>
+            </ul>
+            
+            <div class="depart">
+            <input class="from_destination"type="text" class="depart-field" placeholder="Depart">
+            <input class="datedepart" type="text" class="date-field" placeholder="mm/dd/yy">
+            </div>
+            
+            <div class="arrive">
+            <input class="to_destination"type="text" class="arrive-field" placeholder="Arrive">
+            <input class="datearrive"type="text" class="date-field" placeholder="mm/dd/yy">
+            </div>
+            
+            <div class="passengers">
+                <
+                
+            </div>
+        </section>
+    </main>
+
```

Comment:
Great job using semantic elements and inputs!
Overall you've done a great job with the assignments, and you've been able to complete them. It's definitely something to be proud of, so keep working hard.